environments/stg/global.tfvars

```diff
@@ -49,4 +49,14 @@ frontends = [
       SQLI = ["942440"]
     }
   },
+  {
+    name             = "pet-app1"
+    custom_domain    = "aat.platform.hmcts.net"
+    backend_domain   = ["firewall-prod-int-palo-aat.uksouth.cloudapp.azure.com"]
+    certificate_name = "wildcard-aat-platform-hmcts-net"
+    disabled_rules = {
```

Comment:
do these rules need to be disabled?

Reply:
I will remove them, this is being done temporarily for testing purposes